### PR TITLE
New and fixed

### PR DESCRIPTION
**Pull request by maxmosk**
New viewer (atril) and fixed checking of task number.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ cp okular.sh ~/gnu-koryavov/
 
 Чтобы `gnu-koryavov` запускал Ваш скрипт вместо `okular.sh`, в файле `~/gnu-koryavov/config.conf` измените значение параметра `pdfviewer_script` на путь к вашему скрипту, как это сделано по умолчанию. *Не забудьте поделиться этим скриптом со всеми пользователями `gnu-koryavov`, создав соответсвующий пулл-реквест!*
 
+### Готовые скрипты для просмотра
+- Ocular
```
Comment (sin-diesel) [inline README.md]: Probably just a typo, switch to `Okular`,

File: install.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (sin-diesel) [inline install.sh]: We no longer support bash?

**Review by sin-diesel — CHANGES_REQUESTED** [review]

File: install.sh (hunk 2 above)
Comment (maxmosk) [reply install.sh]: sh exist in all unix-like systems, but usually it is link to bash.

File: README.md (hunk 1 above)
Comment (maxmosk) [reply README.md]: Yes, fixed

File: config.conf [hunk 3]
```diff
@@ -10,4 +10,4 @@ KORYAVNIKS=(
 
 #PDFVIEWER
 
-pdfviewer_script=~/gnu-koryavov/okular.sh
+pdfviewer_script=~/gnu-koryavov/atril.sh
```
Comment (nerett) [inline config.conf]: I think leaving `okular` as a default pdf viewer would be better for most users. For now you can add more information about pdf viewer script selecting to the `README` if you wish

**Review by nerett — CHANGES_REQUESTED** [review]

**Review by sin-diesel — APPROVED** [review]

**Review by nerett — APPROVED** [review]

**Pull request merged**